modules/services/localization/src/main/java/au/com/shiftyjelly/pocketcasts/localization/helper/RelativeDateFormatter.kt

```diff
@@ -43,7 +43,7 @@ class RelativeDateFormatter(val context: Context) {
             }
         } else if (Locale.getDefault().language == "en") {
             // if using an old Android version but using English then add Today and Yesterday
-            val alternativeString = formatCloseDaysOld(calendar, resources)
+            val alternativeString = formatCloseDaysOld(calendar, context.resources)
```

Comment:
I also removed this `resources` param since there's already a field for `context`